design-documents/graph-ql/coverage/b2b/negotiableQuotes.graphqls

```diff
@@ -327,3 +327,7 @@ type NegotiableQuoteUser @doc(description: "A limited view of a Buyer or Seller
     firstname: String!
     lastname: String!
 }
+
+type StoreConfig {
+    is_negotiable_quote_active: Boolean @doc(description: "Indicates if negotiable quote functionality is enabled.")
```

Comment:
Should we expect a `Boolean` value here or a `String`?

Reply:
Boolean preferably.